examples/inverse/plot_morph_stc_to_fsaverage_dipy.py

```diff
@@ -0,0 +1,242 @@
+"""
+==================================================================
+Demonstrate morphing subject source volume to fsaverage using dipy
+==================================================================
+
+This example demonstrates how to morph an individual subject source space to a
+common reference space. For this purpose `dipy
+<http://nipy.org/dipy>`_ will be
+used in order to perform the necessary transforms to a fsaverage serving as
+reference space. The example uses parts of the MNE example
+:ref:`LCMV beamformer pipeline
+<sphx_glr_auto_examples_inverse_plot_lcmv_beamformer_volume.py>`.
+The respective result will be morphed based on an affine transformation and a
+nonlinear morph, estimated based on respective transformation from the
+subject's anatomical T1 (brain) to fsaverage T1 (brain). Afterwards the
+transformation will be applied to the beamformer result. Affine transformations
+are computed based on the mutual information and nonlinear transformations
+will be performed as Symmetric Diffeomorphic Registration using the
+cross-correlation metric [1]_.
+
+.. note:: This example does quite a bit of processing, so even on a
+          fast machine it can take a couple of minutes to complete.
```

Comment:
no reasonable way to speed it up, e.g. by downsampling heavily?

Reply:
well, there are two ways to speed it up "reasonably", maybe also a bit depending on how accurate it has to be in the end. E.g. given that the blobs produced by an inverse solution are normally significantly larger than 1 voxel, we could a) downsample as you suggested (maybe by a factor 3 or even 5) and b) reduce the number of iterations, when doing the registration. I just didn't play around too much with the parameters yet (mostly all defaults from all examples), and as it is in the moment it would be a good working high resolution default. But sure: I'll explore the parameters anyway, so we could decide whether we have a fast or accurate solution (ideally both I suppose ;) )

Reply:
For the examples, we often lean heavily towards performance on that trade-off. Can just put a note up saying, "for real data, you would" blah blah.

Reply:
Your wish is my command! ;)